doc/quickstart/run.rst

```diff
@@ -49,6 +49,40 @@ Possible values are:
   - `default`: fail if there are any errors
   - `strict`: fail if there are any warnings
 
+To re-use pre-processed files from a previous run of the same recipe, you can
+use
+
+.. code:: bash
+
+    esmvaltool run recipe_python.yml --resume_from ~/esmvaltool_output/recipe_python_20210930_123907
+
+Multiple directories can be specified for re-use, make sure to quote them:
+
+.. code:: bash
+
+    esmvaltool run recipe_python.yml --resume_from "~/esmvaltool_output/recipe_python_20210930_101007 ~/esmvaltool_output/recipe_python_20210930_123907"
+
+The first preprocessor directory containing the required data will be used.
+
+This feature can be useful when developing new diagnostics, because it avoids
+the need to re-run the preprocessor.
+Another potential use case is running the preprocessing part of a recipe on
+one or more machines that have access to a lot of data and then running the
+diagnostics on a machine without access to data.
```

Comment:
this doesn't work currently since the machines need to share the path to the preprocessor dir, so that means they are on the same FS, so access to data will be possible. Maybe add a note that it's something we'll have fully working in the future?

Reply:
This should actually work. Have you tried moving the output_dir to a different place after the initial run and then resuming from it?

Reply:
sure it works, I am talking about the e.g. general bit where you run the preproc on JASMIN and you want to run your diag on DKRZ - without scp-ing or other means of moving data

Reply:
Thanks, yes, that would require more work, e.g. setting up a WPS service on ESGF based on ESMValCore for doing the preprocessing and then integrating support for that into ESMValCore. There is something along these lines mentioned in the IS-ENES3 proposal, but let's see if that ever happens or if we're fine rsync'ing data around.